worldedit-core/src/main/java/com/sk89q/worldedit/extension/platform/AbstractPlatform.java

```diff
@@ -84,4 +85,8 @@ public void reload() {
         getTranslationManager().reload();
     }
 
+    @Override
+    public String getId() {
+        return "legacy:" + getPlatformName().toLowerCase(Locale.ROOT).replace(" ", "_");
```

Comment:
need to sub everything outside `[a-z_.-]` with `_` probably, not just spaces, for full transformation.